components/brave_shields/browser/ad_block_subscription_source_provider.h

```diff
@@ -0,0 +1,43 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
```

Comment:
pls check for 2022 everywhere

Reply:
haha, well the unfortunate part is that it was the correct year when this PR was started

Reply:
I think it should actually stay 2021 since that's when the code was written, we don't bump copyright when we make changes